src/containers/Sidebar/partials/SidebarAccordion/SidebarAccordion.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  Box,
+} from "@material-ui/core";
+
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { AppContext } from "./../../../../App";
```

Comment:
project should have relative path `baseUrl: "src"` to avoid this not-so-nice way of importing in components

Reply:
It's relative to the src component. The problem is that it's just nested too deep so I will apply a better structure.